server/src/resolvers/getVariantsResolver/getVariantsResolver.ts

```diff
@@ -65,19 +66,45 @@ const resolveVariantQuery = async (args: QueryInput): Promise<CombinedVariantQue
     }
   });
 
-  // once variants are merged, handle annotations
-  const caddAannotations = settled.find(
+  // filter data that are not in user requested assemblyId
+  const dataForLiftover = combinedResults.filter(v => v.variant.assemblyId !== assemblyId);
+  // filter data that are already in user requested assemlbyId
+  let dataForAnnotation = combinedResults.filter(v => {
+    if (resolveAssembly(v.variant.assemblyId) === assemblyId) {
```

Comment:
Should we standardize the `variant.assemblyId` in the `adapter` files, instead of the `resolver` files? This is to avoid repetitively calling `resolveAssembly` every time we need to standardize the assembly.